### DISCUSSION TITLE
Testing ONNX-Runtime on M1 fails

### DISCUSSION THREAD

**Discussion by AI-Ingrid** [FAST-Imaging/FAST#175] [General]
When attempting to run a custom pipeline with `pyFAST` on M1 chip, it was observed that pyFAST failed to find any inference engines.

We used a wheel provided by @smistad directly, hence, it might have been fixed in newer commits.

```
INFO [0x1e1765b40] Loading inference engines in folder /Users/ikolderu/PycharmProjects/master/venv/lib/python3.9/site-packages/fast/bin//..//lib/
INFO [0x1e1765b40] Loading inference engine ONNXRuntime from shared library /Users/ikolderu/PycharmProjects/master/venv/lib/python3.9/site-packages/fast/bin//..//lib//libInferenceEngineONNXRuntime.dylib
WARNING [0x1e1765b40] Failed to load plugin because dlopen(/Users/ikolderu/PycharmProjects/master/venv/lib/python3.9/site-packages/fast/bin//..//lib//libInferenceEngineONNXRuntime.dylib, 0x0001): Symbol not found: __ZN4fast15InferenceEngine12addInputNodeEjNSt3__112basic_stringIcNS1_11char_traitsIcEENS1_9allocatorIcEEEENS_8NodeTypeENS_11TensorShapeE
  Referenced from: <017433DC-A8C7-3A63-B045-BE0DAA69122E> /Users/ikolderu/PycharmProjects/master/venv/lib/python3.9/site-packages/fast/lib/libInferenceEngineONNXRuntime.dylib
  Expected in:     <33789237-89D3-345A-88B9-1B65417FB59E> /Users/ikolderu/PycharmProjects/master/venv/lib/python3.9/site-packages/fast/lib/libFAST.4.dylib
```

cc: @andreped, @ingridtv

**Comment by smistad** [comment]
I think I may have just built the python wheel incorrectly. I will check tomorrow.

**Reply by smistad** [reply]
I compiled a new wheel which you can try @AI-Ingrid. This fixed this error message for me: 
https://www.dropbox.com/s/mkhmgrf41ucv4sm/pyFAST-4.7.0-cp36-abi3-macosx_11_0_arm64.whl?dl=0

Remember to use --force-reinstall option when installing the wheel as the version number is the same:
`pip3 install --force-reinstall pyFAST-4.7.0-cp36-abi3-macosx_11_0_arm64.whl`

**Reply by AI-Ingrid** [reply]
The provided wheel works! Thank you :D
